src/Lecture2.hs

```diff
@@ -177,8 +202,13 @@ False
 >>> isIncreasing [1 .. 10]
 True
 -}
+
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing [] = True
+isIncreasing [_] = True
+isIncreasing [x,y] = x<=y
+isIncreasing (x:y:xs) = x<=y && isIncreasing xs
```

Comment:
It looks like this implementation returns `True` on the list `[1, 2, 1]` but this list is not increasing